shared-libs/search/src/search.js

```diff
@@ -30,7 +30,7 @@ module.exports = function(Promise, DB) {
       start = options.skip;
       end = start + options.limit;
     }
-    return _.sortBy(rows, 'value').slice(start, end);
+    return _.sortBy(rows, row => 'sort' in row ? row.sort : row.value).slice(start, end);
```

Comment:
If I have:

```Contact A: { dead, muted, value: 'zzz' }
Contact B: { dead, muted, value: 'aaa' }
```

Is contact B guaranteed to appear above contact A if we only sortBy row.sort?

Reply:
Nevermind. The doc's name is in row.sort.